src/content/community/get-involved/index.md

```diff
@@ -99,8 +99,14 @@ The Ethereum ecosystem is on a mission to fund public goods and impactful projec
 
 ## Join a DAO {#decentralized-autonomous-organizations-daos}
 
-"DAOs" are decentralized autonomous organizations. These groups leverage Ethereum technology to facilitate organization and collaboration. For instance, for controlling membership, voting on proposals, or managing pooled assets. While DAOs are still experimental, they offer opportunities for you to find groups that you identify with, find collaborators, and grow your impact on the Ethereum community. [More on DAOs](/dao/)
+"DAOs" are decentralized autonomous organizations. These groups leverage Ethereum technology to facilitate organization and collaboration. For instance, for controlling membership, voting on proposals, or managing pooled assets. While DAOs are still experimental, they offer opportunities for you to find groups that you identify with, find collaborators, and grow your impact on the Ethereum community and the world. [More on DAOs](/dao/)
```

Comment:
Here I'd keep the current description, as this consequence is implicit.

Reply:
I disagree since DAOs arent only about impact on ethereum or crypto. they can be about anything and that fact should be more obvious.

But okay, I've put it back